scripts/test-turbo-modules.sh

```diff
@@ -26,7 +26,7 @@ usage() {
   echo "  -A, --android                      Build for Android."
   echo "  -I, --ios                          Build for iOS."
   echo "  -C, --ubrn-config                  Use a ubrn config file."
-  echo "  -P, --packgage-json-mixin          Merge another JSON file into package.json"
+  echo "  -P, --package-json-mixin          Merge another JSON file into package.json"
```

Comment:
```suggestion
  echo "  -P, --package-json-mixin           Merge another JSON file into package.json"
```

Sorry, I think I have some obsessive compulsive disorder when it comes to alignment. :sweat_smile: